### ISSUE TITLE
oversampling- overfitting

### ISSUE THREAD

**Issue by ClimbsRocks** [ClimbsRocks/auto_ml#280]
we start pretty easily overfitting to our oversampled data. still get pretty good scores on our test set, but definitely overfit to the training set.
